### ISSUE TITLE
AppBarButton renders InputGestureText for primary commands

### ISSUE THREAD

**Issue by wmrutten** [Kinnara/ModernWpf#84]
The rendering of `AppBarButton` controls with an `InputGestureText` in the `CommandBarFlyout.PrimaryCommands` collection seems to be broken.

The `AppBarButton.xaml` control template defines a trigger that displays the `KeyboardAcceleratorTextLabel' when `InputGestureText` is set:

```
<MultiTrigger>
	<MultiTrigger.Conditions>
		<Condition Property="HasInputGestureText" Value="True" />
		<Condition Property="IsInOverflow" Value="True" />
	</MultiTrigger.Conditions>
	<Setter TargetName="KeyboardAcceleratorTextLabel" Property="Visibility" Value="Visible" />
</MultiTrigger>
```

Expected: button label & gesture text are hidden in the primary command collection (only show the button icon).

**Comment by Kinnara** [comment]
Thanks for reporting. Fixed by cecc7d7.

**Comment by wmrutten** [comment]
Thank you!
